testing_and_setup/compass/ocean/drying_slope/forward_idealized_transect.xml

```diff
@@ -0,0 +1,17 @@
+<template>
+	<namelist>
+		<option name="config_Rayleigh_friction">.false.</option>
+		<option name="config_Rayleigh_damping_depth_variable">.false.</option>
+		<option name="config_use_implicit_bottom_drag">.false.</option>
+		<option name="config_use_implicit_bottom_drag_variable">.false.</option>
+		<option name="config_use_implicit_bottom_drag_variable_mannings">.true.</option>
+		<option name="config_thickness_flux_type">'centered'</option>
+		<option name="config_use_idealized_transect">.true.</option>
```

Comment:
`config_use_idealized_transect` is an init mode option right?

Reply:
Yes, thanks for pointing it out.